src/cryptography/hazmat/bindings/openssl/ssl.py

```diff
@@ -438,6 +440,13 @@
 long (*SSL_get_tlsext_status_ocsp_resp)(SSL *, const unsigned char **) = NULL;
 #endif
 
+#ifdef SSL_CTRL_SET_TLSEXT_STATUS_REQ_TYPE
```

Comment:
Could you put a comment that explains what version this was added. It will help us in the future.